### PR TITLE
Added a third graph to unbound with more metrics

### PR DESCRIPTION
**Pull request by hanserasmus**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by hanserasmus** [comment]
Is there something I could do about the failed checks in Travis? Not sure what failed or why, but I'm more than willing to change anything that needs changing from my end.

File: includes/polling/applications/unbound.inc.php [hunk 1]
```diff
@@ -77,6 +77,23 @@
...
+    ->addDataset('rcodeNOERROR', 'DERIVE', 0, 125000000000)
+    ->addDataset('rcodeNXDOMAIN', 'DERIVE', 0, 125000000000)
+    ->addDataset('rcodeNodata', 'DERIVE', 0, 125000000000);
+$fields = array (
+    'opcodeQuery' => $unbound['num.query.opcode.QUERY'],
+    'rcodeNOERROR' => $unbound['num.answer.rcode.NOERROR'],
+    'rcodeNXDOMAIN' => $unbound['num.answer.rcode.NXDOMAIN'],
+    'rcodeNodata' => $unbound['num.answer.rcode.nodata']
+    );
+$metrics['operations'] = $fields;
```
Comment (murrant) [inline includes/polling/applications/unbound.inc.php]: You need to merge the arrays here:
```
$metrics += $fields;
```

Comment (hanserasmus) [reply includes/polling/applications/unbound.inc.php]: Done. Is this correct?

**Comment by f0o** [comment]
Why all the whitespace/newline changes?

File: includes/polling/applications/unbound.inc.php (hunk 1 above)
Comment (murrant) [reply includes/polling/applications/unbound.inc.php]: I was wrong :)

**Pull request merged**